crates/ibc/src/core/ics03_connection/handler/conn_open_init.rs

```diff
@@ -58,6 +58,8 @@ where
         msg.client_id_on_a.clone(),
         Counterparty::new(
             msg.counterparty.client_id().clone(),
+            // Note: the counterparty connection Id passed by the relayer alternated to `None`,
+            // left for the `conn_open_try` process to set
```

Comment:
I don't think we need this comment; we're only following the standard here.

Reply:
[e2ea605](https://github.com/cosmos/ibc-rs/pull/360/commits/e2ea6054fee2be83cd9421fce8b0b21bd92e754b)